packages/kv-store/src/Redis/Config/Scheme.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tempest\KeyValue\Redis\Config;
+
+/**
+ * Represents the connection scheme used to connect to Redis (predis only).
+ */
+enum Scheme: string
```

Comment:
What I don't like is that now there's a naming collision between Smtp Schemes and Redis Schemes. Let's rename to `RedisScheme`